src/main/java/org/folio/oaipmh/helpers/records/RecordMetadataManager.java

```diff
@@ -134,15 +154,22 @@ private void updateFieldsWithItemEffectiveLocationField(JsonObject itemData, Lis
    *
    * @param itemData         - json of single item which contains array of electronic accesses
    * @param marcRecordFields - fields list to be updated with new one
+   * @param suppressedRecordsProcessing - include suppressed flag in 856 field?
    */
-  private void updateFieldsWithItemElectronicAccessField(JsonObject itemData, List<Object> marcRecordFields) {
+  private void updateFieldsWithItemElectronicAccessField(JsonObject itemData,
+                                                         List<Object> marcRecordFields,
+                                                         boolean suppressedRecordsProcessing) {
     JsonArray electronicAccessArray = itemData.getJsonArray(ELECTRONIC_ACCESS);
     if (Objects.nonNull(electronicAccessArray)) {
       electronicAccessArray.forEach(electronicAccess -> {
         Map<String, Object> electronicAccessSubFields = constructElectronicAccessSubFieldsMap((JsonObject) electronicAccess);
         FieldBuilder fieldBuilder = new FieldBuilder();
-
         List<String> indicators = resolveIndicatorsValue((JsonObject) electronicAccess);
+        if (suppressedRecordsProcessing) {
+          final Boolean itemSuppressedFromDiscovery = itemData.getBoolean(INVENTORY_SUPPRESS_DISCOVERY_FIELD);
+          int subFieldValue = itemSuppressedFromDiscovery == null || !itemSuppressedFromDiscovery ? 0 : 1;
```

Comment:
The same here: taking into account that we are already used "apache common lang3" this confditions can be replaced by BooleanUtils.isFalse(). How fo you think, does it make sence?

Reply:
Done.